deploy/ocp-templates/glusters3template.json

```diff
@@ -0,0 +1,205 @@
+{
+    "kind": "Template",
```

Comment:
Can we have the template in yaml please..All the other templates have been ported to yaml for ocp-templates